director/redirect.go

```diff
@@ -522,7 +519,7 @@ func RegisterDirector(router *gin.RouterGroup) {
 	router.GET("/api/v1.0/director/object/*any", RedirectToCache)
 	router.GET("/api/v1.0/director/origin/*any", RedirectToOrigin)
 	router.POST("/api/v1.0/director/registerOrigin", RegisterOrigin)
-	router.GET("/api/v1.0/director/discoverOrigins", DiscoverOrigins)
+	router.GET("/api/v1.0/director/discoverOrigins", DiscoverOriginCache)
```

Comment:
Maybe leave a comments for if we ever make a v2.0 version of the api to change it to `/director/discoverOriginCache`